src/pyDE1/scale/felicita_arc.py

```diff
@@ -0,0 +1,147 @@
+        await super(FelicitaArc, self).standard_initialization(
+            hold_notification=True)
+        if not hold_notification:
+            await self._notify_ready()
+
+    async def update_self_from_device(self):
+        # Not all HW revisions allow for information fetching
+        raise NotImplementedError
+
+    async def start_sending_weight_updates(self):
+        await self._bleak_client.start_notify(
+            Characteristic.MAIN.cuuid,
+            self._create_weight_update_hander())
+        logger.info("Sending weight updates")
+
+    async def stop_sending_weight_updates(self):
+        await self._bleak_client.stop_notify(
+            Characteristic.MAIN.cuuid)
+        logger.info("Stopped weight updates")
+
+    def is_sending_weight_updates(self):
+        return NotImplementedError
+
+    async def _tare_internal(self):
+        await self.send_command(Command.TARE)
+        logger.info("Internal tare sent")
+
+    async def current_weight(self):
+        raise NotImplementedError
+
+    async def display_on(self):
+        raise NotImplementedError
+
+    async def display_off(self):
+        raise NotImplementedError
+
+    async def set_grams(self):
+        raise NotImplementedError
+
+
+    @property
+    def supports_button_press(self):
+        return False
+
+    async def start_sending_button_updates(self):
+        raise NotImplementedError
+
+    async def stop_sending_button_updates(self):
+        raise NotImplementedError
+
+    async def send_command(self, command: "Command"):
+        await self._bleak_client.write_gatt_char(command.cuuid, command.value,
+            response=self._write_gatt_char_response)
+
+    def _create_weight_update_hander(self) -> Callable:
+        scale = self
+
+        async def weight_update_handler(sender, data):
+            nonlocal scale
+
+            if len(data) < 9:
+                return
+
+            now = time.time()
+
+            header1 = data[0]
+            header2 = data[1]
+            
+            sign = data[2]
+            weight = int(data[3:]) / 100.0
+
+
+            if chr(sign) == '-':
+                weight *= -1
+            
+
+            self._update_scale_time_estimator(now)
+
+            await scale.event_weight_update.publish(
+                ScaleWeightUpdate(
+                    arrival_time=now,
+                    scale_time=self._scale_time_from_latest_arrival(now),
+                    weight=weight
+                ))
+
+        return weight_update_handler
+
+
+class Characteristic(enum.Enum):
+
+    MAIN =    'FFE1'  # RWN
+    @property
+    def cuuid(self):
+        return f"0000{self.value}-0000-1000-8000-00805f9b34fb"
+
+
+# These typically get written to CONFIGURATION_EF80
+class Command(enum.Enum):
+    TARE = b'\x54'
+
+    TIMER_ZERO = b'\x43'
+    TIMER_START = b'\x52'
+    TIMER_STOP = b'\x53'
+
+    @property
+    def cuuid(self):
+        return Characteristic.CONFIGURATION_EF80.cuuid
```

Comment:
I'm not sure if the comment and cuuid property are what were expected here.

`send_command` will send to the `cuuid` property of the command.  

I _think_ that this might be Characteristic.MAIN